packages/agents/lighthouse/src/tasks/propagate/helpers/consensys.ts

```diff
@@ -19,7 +19,8 @@ export const getPropagateParams = async (
     l2ChainId,
   });
 
-  const _fee = utils.parseEther("0.01").toString();
+  // the additional optional "postman" fee = 0 currently
+  const _fee = utils.parseEther("0").toString();
```

Comment:
is there a way to pull this dynamically? can you provide a link to where this is documented?

Reply:
Linea team said we can estimate `postman` fee using their SDK, but SDK is not available yet. I couldn't find any doc for that. just from the direct chat with them

Reply:
We can leave this for now and know we have to change it later once the SDK is available.